client/settings/card-body/styles.scss

```diff
@@ -1,17 +1,45 @@
 .wcpay-card-body {
-	> :first-child {
+	&.is-size-medium {
+		// increasing the specificity of the styles to override the Gutenberg ones
+		&#{&} {
```

Comment:
These changes all look good visually and I'm approving, although I'm not certain what this rule is for. This targets an embedded CardBody within another CardBody element? Where is that happening, the UPE Setup banner?

![Screen Shot 2021-07-27 at 4 38 39 PM](https://user-images.githubusercontent.com/6723003/127173874-67f39d91-4969-4e0e-a964-8dde196bd316.png)

Reply:
It increases the specificity of the selector without having to use `!important`, so that it can override the values correctly.
It basically duplicates the selector's name, so that it's more specific

With:
![Screen Shot 2021-07-27 at 9 44 26 AM](https://user-images.githubusercontent.com/273592/127174885-664b0744-40d3-473e-99f3-a22765b14128.png)

Without:
![Screen Shot 2021-07-27 at 9 45 45 AM](https://user-images.githubusercontent.com/273592/127175091-5a7e6674-bb23-48a9-a4f2-f73d9062dc8f.png)

Reply:
Thanks! I'd never seen that before 🕵️‍♂️